qa/tasks/lost_unfound.py

```diff
@@ -7,14 +7,15 @@
 from tasks.util.rados import rados
 from teuthology import misc as teuthology
 from teuthology.orchestra import run
+#import json
```

Comment:
nit: this is not needed

Reply:
It is there if you uncomment the code to get the pg query information, then you also uncomment the "import json" line.

Reply:
removed